spirv_glsl.cpp

```diff
@@ -4401,8 +4413,11 @@ void CompilerGLSL::emit_extension_workarounds(spv::ExecutionModel model)
 		}
 		statement("");
 	}
+}
 
-	if (requires_transpose_2x2)
+void CompilerGLSL::emit_polyfills()
+{
+	if (required_polyfills & PolyfillTranspose2x2)
```

Comment:
Note that there are problems here with mediump vs high. If a highp matrix is used in a fragment shader, it will be demoted to mediump here. See row major load workaround where it needs to stamp out two variants that are highp and mediump.

The existing transpose code is also bugged, so I can fix it up later if it's a hassle.

Reply:
Will fix across the board while I'm at it.  I assume we don't care about `lowp`?

Reply:
No. SPIR-V does not support lowp.